website/src/categories/components/text-field/code.md

```diff
@@ -9,25 +9,21 @@ eleventyNavigation:
 <section class="no-heading">
 
 <div class="ds-example">
-<form>
-  <sl-label for="nickname">Nickname</sl-label>
   <sl-text-field
-    id="nickname"
-    hint="What would you like people to call you?"
+    id="last-name"
+    placeholder="Type your last name"
```

Comment:
I think "type your last name" is a bad example for a placeholder.
Maybe don't use a placeholder, but fill in a value. That will be a way more common scenario.

Reply:
Ok, I'll change it.